src/playbook.yaml

```diff
@@ -95,3 +95,11 @@
 
     - name: Install BiocManager
       shell: "R -e \"install.packages('BiocManager')\""
+
+    - name: update limits.conf
+      shell:
+        cmd:
```

Comment:
hmm, it looks like other shell commands in this file do not contain `cmd`.

`cmd` is part of the `ansible.builtin.shell` target.  are you sure `shell` is the same as `ansible.builtin.shell`?

Reply:
@zaro0508 I'm following exactly the pattern used at line [41](https://github.com/Sage-Bionetworks-IT/packer-rstudio/blob/master/src/playbook.yaml#L41).

Reply:
i think you are missing the pipe?  like this `cmd: |` ?

Reply:
Fixed!